src/sqlfluff/core/rules/std/L020.py

```diff
@@ -1,10 +1,26 @@
 """Implementation of Rule L020."""
 
 import itertools
+from typing import List, NamedTuple, Optional
 
+from sqlfluff.core.dialects.base import Dialect
+from sqlfluff.core.dialects.dialect_ansi import AliasInfo
+from sqlfluff.core.parser.segments.base import BaseSegment
 from sqlfluff.core.rules.base import BaseCrawler, LintResult
 
 
+class SelectStatementColumnsAndTables(NamedTuple):
```

Comment:
I refactored `L020.py` to make its `SELECT` analysis reusable without having to inherit from it. No change in functionality.